vpr/src/device/rr_graph_view.h

```diff
@@ -62,6 +62,16 @@ class RRGraphView {
         return node_storage_.node_capacity(node);
     }
 
+    /* Get the direction of a routing resource node. This function is inlined for runtime optimization. */
```

Comment:
This is a good time to beef up the comment.  Explain what the four direction types are, either here or in the vpr_types.h where the enum is defined (or both).  
INC_DIRECTION: wire driver is positioned at the low-coordinate end of the wire.
DEC_DIRECTION: wire_driver is positioned at the high-coordinate end of the wire.
BI_DIRECTION: wire has multiple drivers, so signals can travel either way along the wire
NO_DIRECTION: wire does not have a valid direction set 

I also think it would be good to change NO_DIRECTION to INVALID_DIRECTION to make its meaning more clear.

Reply:
I have added the comments here and in `vpr_types.h`. I have also changed `NO_DIRECTION` to `INVALID_DIRECTION`. I will wait until Xifan and Henner give their opinions about the C-strings v. C++ strings until I do anything there.